app/views/plan/features.scala

```diff
@@ -78,7 +78,7 @@ object features {
             ),
             tr(unlimited)(
               a(href := routes.Search.index(1))("Advanced search"),
-              " through more than 700 thousand Lishogi games"
+              " through more than 800 thousand Lishogi games"
```

Comment:
Normally I'd suggest a variable here, but as you point out once 1 million is crossed this will change again.